tests/functionaltests/Tests/UIKitTests/UIButtonTests.mm

```diff
@@ -36,6 +36,72 @@
     ASSERT_TRUE([backingElement isKindOfClass:[WXFrameworkElement class]]);
 }
 
+@interface UIDeallocTestButton : UIButton {
+    NSCondition* _condition;
+    BOOL* _signaled;
```

Comment:
My bad, I see the usage now.

---
In reply to: [98299648](https://github.com/Microsoft/WinObjC/pull/1861#discussion_r98299648) [](ancestors = 98299648,98299381)